static/js/stream_list.js

```diff
@@ -153,14 +184,15 @@ exports.set_in_home_view = function (stream, in_home) {
     }
 };
 
-function build_stream_sidebar_row(name) {
+function build_stream_sidebar_row (name) {
```

Comment:
You added an inadvertent space here. It's a good habit to go through and review the diff yourself before pushing. I usually just run `git diff` in the console and look through it before I create a PR